src/navigation/RootNavigator.tsx

```diff
@@ -75,22 +78,186 @@ export default function RootNavigator() {
             <Stack.Screen
               name="TabsStack"
               component={TabsStack}
-              options={{ headerShown: false }}
+              options={({ navigation }) => ({
+                headerTitle: () => {
+                  return <LogoTitle />;
+                },
+                headerTitleAlign: 'center',
+                headerStyle: {
+                  backgroundColor: '#CFD3D7',
+                  height: 96,
+                  borderBottomLeftRadius: 5,
+                  borderBottomRightRadius: 5,
+                },
+                headerLeft: () => (
+                  <MaterialCommunityIcons
+                    name="cog"
+                    color={Colors.brandBlue}
+                    size={20}
+                    style={{
+                      position: 'absolute',
+                      left: '8.21%',
```

Comment:
[NIT (nitpicky)] Is there a reason you're using such precise percentages? Try to round to nearest whole / half / quarter if precision isn't absolutely necessary. Small thing, but keeps it simple for readers of your code :)

Reply:
I was just using what the Figma file described, but I can totally round to the nearest whole if that is preferred (:

Reply:
ah that makes sense! Figma dimensions aren't a hard rule since implementations don't always look the same and cases like this where an .21 probably doesn't matter ✨✨